src/components/Details/SettingsComponent.tsx

```diff
@@ -120,6 +120,7 @@ export const Settings = () => {
   }
 
   function onRefresh() {
+    console.log("refreshed");
```

Comment:
Avoid leaving in too many debugging console.log in the code, or at least use console.debug. Console.log should only be used for important runtime information.